.changeset/yellow-dots-listen.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/blocks-admin": minor
+---
+
+Change headline from "Columns" to "Items" / Only show a select box when there are multiple layouts available
```

Comment:
This is a little vague... this will be shown in the general Comet Release-Notes (e.g.: https://github.com/vivid-planet/comet/releases/tag/v4.6.0) so it should also mention where the change is made. 

The details of this change are probably not too important anyway when updating the Comet version, so maybe it's enough to write "UI improvements in admin of `ColumnsBlock`"